CodeStyle.Analyzers.Test/CodeStyle.Analyzers.Test.csproj

```
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>netcoreapp2.0</TargetFramework>
```

I'd target the newest framework for the tests
```suggestion
    <TargetFramework>net5.0</TargetFramework>
```